.travis.yml

```diff
@@ -7,9 +7,14 @@ addons:
     packages:
     - bzr
 
+env:
+  global:
+    - PIP_UNSTABLE_FEATURE=
+
```

Comment:
No need to add this?

Reply:
This can be removed.

Reply:
This is me trying various things and [doc says an env needs to be present in the top level](https://docs.travis-ci.com/user/customizing-the-build/#matching-jobs-with-allow_failures). (And apprarently it’s not enough.)